### DISCUSSION TITLE
Alguma forma de trabalhar em empresa/projeto voluntário ?

### DISCUSSION THREAD

**Discussion by MatheusNtg** [frontendbr/forum#897] [Discussão]
Estudo front-end a um tempo e gostaria de praticar oque aprendi até agora, porém creio que oque sei não é o suficiente para começar a trabalhar (remuneradamente) com isso. Gostaria de saber se alguém conhece alguma forma de trabalhar remotamente (e voluntariamente) para alguma empresa ou algum projeto (que não seja aqui no GitHub pq até agora não encontrei um repo para o meu nível) para assim adquirir experiência.

**Comment by felipemfp** [comment]
Aqui mesmo temos um [repositório](https://github.com/frontendbr/open-source) de projetos que gente como a gente pedem ajuda, geralmente com frontend.

Esse é outro [repostório](https://github.com/LFeh/frontend-challenges) bacana com desafios propostos por empresas em processos seletivos: https://github.com/LFeh/frontend-challenges.

**Comment by luizbills** [comment]
> ...porém creio que oque sei não é o suficiente para começar a trabalhar (remuneradamente) com isso.

Só uma dica: é trabalhando que você vai ter mais motivação e criar experiência para o mercado. Então não fica pensando que você não está pronto. Dá a cara a tapa logo. Procura uns freelas e vai indo.

**Comment by felipefialho** [comment]
Como dito pelo @felipemfp, o [Challenges](https://github.com/LFeh/frontend-challenges) é um lugar para testar seus conhecimentos e para melhorar ainda mais, participar de projetos open-source é a melhor decisão que você pode tomar.

Algumas issues que podem te ajudar: #801 #5 #456

**Comment by onaSousa** [comment]
@MatheusNtg contribuir com os repos do grupo ou outros aqui no Github ajuda muito e provável que vc seja até ajudado caso sua contribuição não esteja bem de acordo com o repositório. 
Além do Github eu indicaria o site www.atados.com.br, lá tem muita demanda pra criação de sites institucionais caso você já esteja mais seguro na criação de sites e queira ajudar mais gente e incluir no seu protfolio.

**Comment by leandroruel** [comment]
@onaSousa esse atados funciona? ta em branco aqui rs

**Comment by Morette** [comment]
@leandroruel 
o atados abriu aqui, demorou um pouco, mas abriu...

sobre projetos open source, o @eduardojmatos criou a excelente Wiki
https://github.com/frontendbr/forum/wiki/Projetos-open-source

**Comment by leandroruel** [comment]
@Morette testei com mais pessoas e não abre por nada, tá travadíssimo.
